File: code-examples/java/promotions/list.java
HttpResponse<String> response = Unirest.get("http://api.marketcloud.it/v0/promotions")
  .header("authorization", "YourPublicKey:YourToken")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .asString();

File: code-examples/csharp/promotions/list.cs
var client = new RestClient("http://api.marketcloud.it/v0/promotions");
var request = new RestRequest(Method.GET);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
IRestResponse response = client.Execute(request);